README.md

```diff
@@ -161,16 +161,22 @@ Babel has many transform options.
 
 `espower-babel` read `${cwd}/.babelrc` if exists.
 
-also can manually configure babel transform options.
+Also, you can manually configure babel transform options.
 
 ```js
 require('espower-babel')({
     babelrc: {
-        stage: 0
+        "presets": ["es2015", "stage-3"],
```

Comment:
`presets` might be different among users so I've [omitted the presets part](https://github.com/power-assert-js/babel-plugin-espower#via-babel-require-hook) in README.